nixos/lib/test-driver/test-driver.py

```diff
@@ -115,7 +117,8 @@ def create_vlan(vlan_nr: str) -> Tuple[str, str, "subprocess.Popen[bytes]", Any]
     fd.write("version\n")
     # TODO: perl version checks if this can be read from
     # an if not, dies. we could hang here forever. Fix it.
-    vde_process.stdout.readline()
+    if vde_process.stdout:
```

Comment:
we really want to die if this is `None`, so an assertion would not hurt.

Reply:
Yes. Just add `assert vde_process.stdout is not None`. Mypy will than know that stdout is never None.

Reply:
will do.